### PR TITLE
Inform user about not supported java version

### PR DESCRIPTION
**Pull request by azdrojowa123**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -146,6 +146,17 @@ allprojects {
     mavenCentral()
   }
 
+  def checkJavaVersion = { task ->
+    if(JavaVersion.current() != JavaVersion.VERSION_11){
+       throw new GradleException("Project must be build with version 11")
```
Comment (PawelLipski) [inline build.gradle:151]:
```suggestion
       throw new GradleException("Project must be built with Java version 11")
```

File: build.gradle [hunk 2]
```diff
@@ -146,6 +146,17 @@ allprojects {
...
+    }
+  }
+
+  afterEvaluate {
+    for(def task in it.tasks.build)
```
Comment (PawelLipski) [inline build.gradle:156]: What tasks are included under `it.tasks.build`... does it correspond to all dependencies of `build` task? 🤔

**Comment by mkondratek** [comment]
on the other hand... ✋ can't we simply make the project compatible with a newer java?

File: build.gradle (hunk 2 above)
Comment (azdrojowa123) [reply build.gradle:156]: it.tasks.build correspond to each module build task. So it doesn't matter if we trigger the build task from whole plugin or from another Gradle module inside them -> checkJavaVersion will be call

**Review by PawelLipski — APPROVED** [review]

**Pull request merged**